docs/sections/catair-mapping.html

```diff
@@ -0,0 +1,50 @@
+<section id="catair-mapping">
+  <h2>Mapping to Legacy Systems</h2>
+
+  <p>Verfiable Credentials are often formed from, or delivered to systems which do not support JSON-LD.</p>
+
+  <p>For example:</p>
+
+  <ul>
+    <li>
+      <a href="https://www.cbp.gov/trade/ace/catair">ACE Automated Broker Interface (ABI) CBP and Trade Automated Interface Requirements (CATAIR)</a>
+    </li>
+  </ul>
+
+
+  <p>
+    This table shows how key data elements of a 
+    <a href="#MillTestReportCredential">Mill Test Report Credential</a>
+    can be related to existing CATAIR documents, 
+    and how certain data elements might be new, 
+    and will have no mapping to CATAIR.
+  </p>
+
+  <table class="simple" >
+    <thead>
+      <tr>
+        <td>JSON Pointer</td>
+        <td>Value</td>
+        <td>CATAIR</td>
```

Comment:
Too detailed. Let's break this down before getting into attribute-level.

Reply:
The original request was "per attribute", for "each credential".

Reply:
I see, I misunderstood. Then should we move this to the Credential section instead. That would make it clearer.